takari-lifecycle-plugin/src/main/java/io/takari/maven/plugins/compile/jdt/classpath/Classpath.java

```diff
@@ -117,7 +115,7 @@ public void reset() {
     return entries;
   }
 
-  public Set<String> getReferencedEntries() {
+  public Set<Path> getReferencedEntries() {
     return referencedentries;
```

Comment:
Would it make sense to use `File` here to have parity with `File` in `AbstractCompiler`?

Reply:
Currently `Classpath` uses `Path`, and `CompilerJdt` translates between `Path` and `File`. Beware, however, that under java 9 system libraries will not have corresponding `File`s